server/index.ts

```diff
@@ -115,7 +115,7 @@ app.use(
 			reportOnly: true,
 			directives: {
 				'connect-src': [
-					MODE === 'development' ? 'ws:' : null,
+					...(MODE === 'development' ? ['ws:', 'localhost:24678'] : []),
```

Comment:
Is there any other way for us to determine this websocket address? The problem is, what happens if I try to run my app two times at once?